dtcli/scripts/dt.py

```diff
@@ -106,6 +107,26 @@ def _gendevcert(
         dev_passphrase
     )
 
+def token_load(ctx, param, value):
+    try:
+        if value == '-':
+            value = DEFAULT_TOKEN_PATH
+
+        with open(value) as f:
+            try:
+                token = f.readlines()[0].rstrip()
+            except IndexError:
+                raise Exception("Token file is empty")
+        return token
+    except FileNotFoundError:

```

Comment:
These errors should translate to Click errors

Reply:
Changed Exception into clickException and add click.BadArgumentUsage if no env var exists